tools/invoke-tests

```diff
@@ -52,5 +52,10 @@ export OS_AUTOINST_BUILD_DIRECTORY=$build_directory
 export OS_AUTOINST_MAKE_TOOL=$make_path
 
 # invoke tests within the t and xt directory
-cd t && "$prove_path" $PROVE_ARGS ${TESTS:-'-r'}
-cd ../xt && "$prove_path" $PROVE_ARGS ${TESTS:-'-r'}
+if [[ $TESTS = *[!\ ]* ]]
+then
+	"$prove_path" $PROVE_ARGS $TESTS
```

Comment:
Don't mix tabs and spaces. This file uses 4 spaces for an indentation level so far so let's keep it this way.

Reply:
I noticed that too, fixed

Reply:
Sorry about that! wrong `.vimrc` :sweat:
Thank you for taking care of that @okurz